### ISSUE TITLE
mocks.dart get error due to HiveInterface signature is changed

### ISSUE THREAD

**Issue by huang12zheng** [flutterdata/flutter_data#155]
git now
```dart
@override
void init(String path) {}
@override
  Future<Box<E>> openBox<E>(
    String name, {
    HiveCipher? encryptionCipher,
    KeyComparator keyComparator = defaultKeyComparator,
    CompactionStrategy compactionStrategy = defaultCompactionStrategy,
    bool crashRecovery = true,
    String? path,
    Uint8List? bytes,
    @Deprecated('Use encryptionCipher instead') List<int>? encryptionKey,
  }) async {
    final box = FakeBox<E>();
    _boxes[name] = box;
    return box;
  }
```
---
my local
```dart
@override
  void init(
    String? path, {
    HiveStorageBackendPreference backendPreference = HiveStorageBackendPreference.native,
  }) {}

@override
  Future<Box<E>> openBox<E>(
    String name, {
    HiveCipher? encryptionCipher,
    KeyComparator keyComparator = defaultKeyComparator,
    CompactionStrategy compactionStrategy = defaultCompactionStrategy,
    bool crashRecovery = true,
    String? path,
    Uint8List? bytes,
    String? collection,
    List<int>? encryptionKey,
  }) async {
    final box = FakeBox<E>();
    _boxes[name] = box;
    return box;
  }
```

**Comment by frank06** [comment]
What exact version of Hive are you using?

**Comment by frank06** [comment]
It's fixed now, you can use `flutter_data: ^1.4.0`
